task_manager/spec/helpers/sessions_helper_spec.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe SessionsHelper, type: :helper do
+  describe 'method' do
+    describe 'log_in' do
+      let!(:user) { create(:user) }
+
+      context 'user情報を入れる' do
+        before { log_in(user) }
+        it { expect(current_user.id).to eq user.id }
+      end
+    end
+
+    describe 'log_out' do
+      let!(:user) { create(:user) }
+      let!(:rspec_session) { { user_id: user.id } }
+      before { log_out }
+
+      context '正常な動作' do
+        it { expect(current_user).to eq nil }
+      end
+    end
+
+    describe 'current_user' do
+      let(:user) { create(:user) }
+      let!(:rspec_session) { { user_id: user.id } }
+      subject { current_user }
+
+      context 'ログイン時' do
+        it { expect(is_expected.to(eq user)) }
```

Comment:
ここだけやこしいことになってるので、suggestion出しました！他は良さそうです！
```suggestion
        it { is_expected.to eq user }
```

Reply:
ありがとうございます！
approveさせていただきました！
pullして、rubocopとテストの確認も行いました！